econml/policy/_forest/_forest.py

```diff
@@ -467,6 +504,9 @@ def predict(self, X):
         Returns
         -------
         treatment : array-like of shape (n_samples)
-            The recommded treatment, i.e. the treatment index with the largest reward for each sample
+            The recommded treatment, i.e. the treatment index with the largest expected reward for each sample.

```

Comment:
Not sure about phrasing...  Maybe something more like 
```suggestion
            The recommded treatment, i.e. the treatment index most often predicted to have the highest reward for each sample.
```